README.md

```diff
@@ -77,7 +77,7 @@ Implemented via rule-based methods (binary projection profile entropy maximizati
 ### Method Behaviour 
 For each page, this processor takes a document image as input and computes the border around the page content area (i.e. removes textual noise as well as any other noise around the page frame). It also annotates a cropped image.
 
-The input image need not be binarized, but should be deskewed for the module to work optimally.
+The input image does not need to be binarized, but should be deskewed for the module to work optimally.
```

Comment:
I've used `need` as a modal verb here. You are probably not aware of that construction.

Reply:
Fine. I now learned that it is even a semi-modal verb and the original form is fine, although at least one comment states _First of all, this is somewhat archaic usage that is found in modern English only when there is a need to sound poetic. In modern English, "needs not be" = "doesn't need to be"_. If you prefer being archaic and sound poetic, we can keep that line.

Reply:
The archaic/poetic attribution is questionable. Whoever wrote this does not read much. (They also _misspelled_ their explanation, it's `need not be` without inflection-s.) As for "Modern English", there has been no change in the last 200 years, and it's still as frequent as `must not` – see [here](https://books.google.com/ngrams/graph?content=need+not%2Cshould+not%2Cmust+not%2Cshall+not%2Cneed+be%2Cneed+not+be&year_start=1800&year_end=2019&corpus=26&smoothing=0) if you believe in statistics.